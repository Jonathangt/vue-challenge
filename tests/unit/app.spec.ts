import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import HomeView from '../../src/views/HomeView.vue'

describe('HomeView', () => {
    it('Matches snapshot', () => {
        const wrapper = mount(HomeView, {})
        expect(wrapper.html()).toMatchSnapshot()
    })
})
