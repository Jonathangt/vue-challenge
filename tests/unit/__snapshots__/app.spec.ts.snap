// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView > Matches snapshot 1`] = `
"<!-- <main>
    <TheWelcome />
  </main> -->"
`;
